cms/envs/common.py

```diff
@@ -155,7 +155,10 @@
     'DASHBOARD_SHARE_SETTINGS': {
         # Note: Ensure 'CUSTOM_COURSE_URLS' has a matching value in lms/envs/common.py
         'CUSTOM_COURSE_URLS': False
-    }
+    },
+
+    # Teams feature
+    'ENABLE_TEAMS': False,
```

Comment:
Instead of putting the value explicitly in both CMS and LMS, can you import it from LMS (see line 36)?